config/routes.rb

```diff
@@ -4,6 +4,8 @@
   devise_for :users # , ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
 
+  get 'nav/:page', to: 'home#nav', as: :navigate
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.